operator/roles/ansible-service-broker/tasks/main.yml

```diff
@@ -5,6 +5,31 @@
   set_fact:
     api_groups: "{{ lookup('k8s', cluster_info='api_groups') }}"
 
+# reconciled_generation - the last reconciled spec generation of a given CR
+# generation - the current generation of a given CR
+# These values are useful because if the current generation value is different
+# than the reconciled_generation value, we know that the CR's spec has been
+# changed, requiring a bounce of the broker's pod to accept new configuration.
+- name: Set reconciled_generation and generation facts
```

Comment:
Disclaimer: I think this comment is a good candidate for a follow-up PR.

We can put these in `vars/main.yaml` and have the first task be:

```
- name: Set important facts before execution
  set_fact:
    api_groups: "{{ lookup('k8s', cluster_info='api_groups') }}"
    pending_config_changes: "{{ generation != reconciled_generation }}"
```